packages/nodes-base/nodes/ConvertKit/ConvertKit.node.ts

```diff
@@ -0,0 +1,287 @@
+			// Form, Sequence, and Tag Operations
+			//--------------------------------------------
+			} else if(['form', 'sequence', 'tag'].includes(resource)) {
+				//-----------------
+				// Add Subscriber
+				//-----------------
+				if(operation === 'addSubscriber') {
+					qs.email= this.getNodeParameter('email', i) as string;
+					const id = this.getNodeParameter('id', i);
+
+					const additionalParams = this.getNodeParameter('additionalFields', 0) as IDataObject;
+
+					if(additionalParams.firstName) {
+						qs.first_name = additionalParams.firstName;
+					}
+
+					if(additionalParams.fields !== undefined) {
+						const fields = {} as IDataObject;
+
+						// @ts-ignore
+						for(let j = 0; j < additionalParams.fields.field.length; j++) {
+							// @ts-ignore
+							const key = additionalParams.fields.field[j].key as string;
+							// @ts-ignore
+							const value = additionalParams.fields.field[j].value;
+
+							fields[key] = value;
+						}
+
+						qs.fields = fields;
+					}
+
+					if(resource === 'form') {
+						method = 'POST';
+						endpoint = `/forms/${id}/subscribe`;
+					} else if(resource === 'sequence') {
+						method = 'POST';
+						endpoint = `/sequences/${id}/subscribe`;
+					} else if(resource === 'tag') {
+						method = 'POST';
+						endpoint = `/tags/${id}/subscribe`;
+					}
+				//-----------------
+				// Get All
+				//-----------------
+				} else if(operation === 'getAll') {
+					method = 'GET';
+					if(resource === 'form') {
+						endpoint = `/forms`;
+					} else if(resource === 'tag') {
+						endpoint = `/tags`;
+					} else if(resource === 'sequence') {
+						endpoint = '/sequences';
+					}
+				//--------------------
+				// Get Subscriptions
+				//--------------------
+				} else if(operation === 'getSubscriptions') {
+					const id = this.getNodeParameter('id', i);
+					const additionalParams = this.getNodeParameter('additionalFields', 0) as IDataObject;
+					if(additionalParams.subscriberState) {
+						qs.subscriber_state = additionalParams.subscriberState;
+					}
+
+					method = 'GET';
+					if(resource === 'form') {
+						//TODO make sure this is working
+						endpoint = `/forms/${id}/subscriptions`;
+					} else if(resource === 'tag') {
+						endpoint = `/tags/${id}/subscriptions`;
+					} else if(resource === 'sequence') {
+						endpoint = `/sequences/${id}/subscriptions`;
+					}
+				//------------
+				// Create Tag
+				//------------
+				} else if(operation === 'create') {
+					const name = this.getNodeParameter('name', i);
+					qs.tag = { name, };
+
+					method = 'POST';
+					endpoint = '/tags';
+				//------------
+				// Remove Tag
+				//------------
+				} else if(operation === 'removeSubscriber') {
+					const id = this.getNodeParameter('id', i);
+
+					qs.email = this.getNodeParameter('email', i);
+
+					method = 'POST';
+					endpoint = `/tags/${id}/unsubscribe`;
+				}
+			} else {
+				throw new Error('Unrecognized resource');
```

Comment:
I like your version, but I think the codebase uses `throw new Error(`The resource "${resource}" is not known!`);`